Source/WarpX.cpp

```diff
@@ -548,6 +561,34 @@ WarpX::ReadParameters ()
             }
         }
 
+	std::vector<int> signals_in;
+	pp_warpx.queryarr("break_signals", signals_in);
```

Comment:
I'll use [`strsignal`](https://man7.org/linux/man-pages/man3/strsignal.3.html) to go from symbolic names like SIGINT or USR1 to the numbers